tests/slt/join_left_inner.slt

```diff
@@ -0,0 +1,93 @@
+statement ok
+create table x(id int primary key, a int, b int);
+
+statement ok
+create table y(id int primary key, c int, d int);
+
+statement ok
+insert into x values (0, 1, 2), (1, 1, 3);
+
+query IIII
+select a, b, c, d from x join y on a = c;
+----
+
+statement ok
+insert into y values (0, 1, 5), (1, 1, 6), (2, 2, 7);
+
+query IIII
+select a, b, c, d from x join y on a = c;
```

Comment:
The file name is join_left_inner, why you test inner join here?

Reply:
> The file name is join_left_inner, why you test inner join here?

for risinglight
https://github.com/risinglightdb/risinglight/blob/main/tests/sql/join_left_inner.slt